daprdocs/content/en/getting-started/quickstarts/actors-quickstart.md

```diff
@@ -0,0 +1,178 @@
+---
+type: docs
+title: "Quickstart: Actors"
+linkTitle: "Actors"
+weight: 75
+description: "Get started with Dapr's Actors building block"
+---
+
+Let's take a look at Dapr's [Actors building block]({{< ref actors >}}). In this Quickstart, you will run a `SmartDevice.Service` microservice and a simple console client to demonstrate the stateful object patterns in Dapr Actors.  
+1. Using a `SmartDevice.Service` microservice, you can host:
+   - Two `SmartDectectorActor` smoke alarm objects
+   - A `ControllerActor` object that commands and controls the smart devices  
+1. Using a `SmartDevice.Client` console app, the client app interacts with each actor, or the controller, to perform actions in aggregate. 
+1. The `SmartDevice.Interfaces` contains the shared interfaces and data types used by both the service and client apps.
+
+<img src="/images/bindings-quickstart/bindings-quickstart.png" width=800 style="padding-bottom:15px;">
```

Comment:
Need a new diagram drawn here which shows 2 device actors and one controller actor. On each actor show the method endpoint and then shows the calls between the actors with numbers. These numbers should follow the same call sequence as describe in the "When you ran the client app:" section below.